docs/user-guide/story-setting.md

```diff
@@ -12,11 +13,19 @@ The editor can enable or disable and edit the following components:
 
 <img src="../img/story-setting/theme.gif" class="ms-docimage"/>
 
+* The **Overlay** to choose the default background and text color of the *Content* present in the [Title Section](title-section.md#content) and in the [Immersive Section](immersive-section.md#content): clicking on the *Change Color* button <img src="../img/button/color-picker.jpg" class="ms-docbutton"/> a color picker appears to allow selecting the desired color:
```

Comment:
* The **Overlay** to choose the default background and text color of *Contents* present in the [Title Section](title-section.md#content) and in the [Immersive Section](immersive-section.md#content) as well: clicking on the *Change Color* button <img src="../img/button/color-picker.jpg" class="ms-docbutton"/> a color picker appears to allow selecting the desired color: